WooCommerce/Classes/Authentication/Epilogue/StorePickerViewController.swift

```diff
@@ -291,6 +291,15 @@ private extension StorePickerViewController {
             return
         }
 
+        // If a site address was passed in credentials, select it
+        // TODO: improve it somehow.
+        //
+        let siteAddress = StoresManager.shared.sessionManager.defaultCredentials?.siteAddress
+        for site in sites where site.url == siteAddress {
+            currentlySelectedSite = site
+            return
+        }
+
```

Comment:
This is comparing two strings instead of two formatted URLs. Is that considered bad practice?

Reply:
Meh. I think its fine.